src/gen.cpp

```diff
@@ -301,10 +301,16 @@ int generate()
     for(int i=0; i<4; i++)
     for(int j=0; j<4; j++)
      pipp += momArray[i]*momArray[j]*gmumu[i]*gmumu[j]*surf[iel].pi[index44(i,j)] ;
-    WviscFactor = (1.0 + (1.0+stat*feq)*pipp/(2.*surf[iel].T*surf[iel].T*(params::ecrit*1.15))) ;
-    if(WviscFactor<0.1) WviscFactor = 0.1 ; // test, jul17; before: 0.5
-    //if(WviscFactor>1.2) WviscFactor = 1.2 ; //              before: 1.5
+    WviscFactor += (1.0+stat*feq)*pipp/(2.*surf[iel].T*surf[iel].T*(params::ecrit+params::ecrit*params::ratio_pressure_energydensity)) ;
+   }
```

Comment:
Indentation after line break could be larger to increase readability

Reply:
I recommend reformatting the codebase as a whole in  a future release, and enforce readable formatting in the future instead.